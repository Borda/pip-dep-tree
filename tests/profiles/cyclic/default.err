Warning!! Cyclic dependencies found:
* CircularDependencyB => CircularDependencyA => CircularDependencyB
* CircularDependencyA => CircularDependencyB => CircularDependencyA
------------------------------------------------------------------------
