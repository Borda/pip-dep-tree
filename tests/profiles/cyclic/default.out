pipdeptree==2.0.0b1
  - pip [required: >=6.0.0, installed: 20.2.3]
setuptools==50.3.0
wheel==0.35.1
